services/cd-service/pkg/service/git.go

```diff
@@ -126,6 +126,10 @@ func (s *GitServer) GetProductSummary(ctx context.Context, in *api.GetProductSum
 }
 
 func (s *GitServer) GetCommitInfo(ctx context.Context, in *api.GetCommitInfoRequest) (*api.GetCommitInfoResponse, error) {
+	if !s.Config.WriteCommitData {
```

Comment:
add missing test for the response code here.
This should be relative easy to test in git_test.go (see TestGetCommitInfo)